2019-09-04 13:59:33,924 ERROR: 'dict' object has no attribute 'codigo' [in /home/cavalcantigor/workspace/python/flaskeleton-api/app/resources/__init__.py:39]
2019-09-04 14:00:04,971 ERROR: 'dict' object has no attribute 'codigo' [in /home/cavalcantigor/workspace/python/flaskeleton-api/app/resources/__init__.py:39]
2019-09-04 14:00:07,693 ERROR: 'dict' object has no attribute 'codigo' [in /home/cavalcantigor/workspace/python/flaskeleton-api/app/resources/__init__.py:39]
2019-09-04 14:00:39,865 ERROR: 'dict' object has no attribute 'codigo' [in /home/cavalcantigor/workspace/python/flaskeleton-api/app/resources/__init__.py:39]
2019-09-04 14:00:39,881 ERROR: 'dict' object has no attribute 'codigo' [in /home/cavalcantigor/workspace/python/flaskeleton-api/app/resources/__init__.py:39]
2019-09-04 14:01:10,561 ERROR: 'Aluno' object is not iterable [in /home/cavalcantigor/workspace/python/flaskeleton-api/app/resources/__init__.py:39]
